components/collaboration/card-components/CollaborationDetails.tsx

```
@@ -0,0 +1,97 @@
+      </View>
+      <View
+        style={{
+          flexDirection: "row",
+          marginTop: 10,
+          flexWrap: "wrap",
+        }}
+      >
+        {contentType &&
+          contentType.map((content, index) => (
+            <ChipCard key={index} chipText={content} chipIcon={faCoins} />
+          ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)